README.md

```diff
@@ -13,7 +13,7 @@ Open terminal and run this command ruby class.rb
 
 ## Author
 
-👤 **Nonhlanhla Mndebele**
+👤 **Nonhlanhla R Mndebele**
 
 - GitHub: [@Nonhlanhla](https://https://github.com/29td)
```

Comment:
Great job @29td 🤝

- [ ] Kindly fix your GitHub handle. The link seems to be broken.